jest.config.js

```diff
@@ -5,7 +5,7 @@ const path = require( 'path' );
 
 module.exports = {
 	preset: '@automattic/calypso-build',
-	roots: [ '<rootDir>/extensions/' ],
+	roots: [ '<rootDir>/extensions/', '<rootDir>/modules/search/instant-search' ],
```

Comment:
Possibly should split this out - this ends up being run by `yarn test-extensions`.

Reply:
Created `yarn test-search`.